content/en/security/cloud_siem/content_packs.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Content Packs
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "/security/cloud_siem/log_detection_rules"
+  tag: "Documentation"
+  text: "Create log detection rules"
+- link: "security/cloud_siem/investigator"
+  tag: "Documentation"
+  text: "Learn more about the Investigator"
+- link: "/security/cloud_siem/investigate_security_signals"
+  tag: "Documentation"
+  text: "Investigate security signals"
+---
+
+## Overview
+
+[Cloud SIEM Content Packs][1] provide out-of-the box content for key security integrations. Depending on the integration, a Content Pack can include the following:
+
+- [Detection Rules][2] to provide comprehensive coverage of your environment
```

Comment:
add:
- [Pipeline][x] to parse and normalize the logs

Reply:
@jnhunsberger which doc would we link to for "Pipeline"? Also, is this already available? I'm not seeing it listed for any of the content packs.